tests/unit/test_charm.py

```diff
@@ -217,11 +249,17 @@ def test_given_service_not_yet_started_when_on_start_then_srsenb_service_is_star
             "systemctl start srsenb", shell=True, stdout=-1, encoding="utf-8"
         )
 
+    @patch("charm.service_active")
     @patch("subprocess.run")
-    def test_given_service_started_when_on_start_then_srsenb_status_is_active(self, _):
+    def test_given_service_started_when_on_start_then_srsenb_status_is_active(
+        self, _, patch_service_active
+    ):
+        self.harness.charm._stored.installed = True
```

Comment:
In places where you dont have to fix the `_stored` values (like here), please don't do it as we'll probably want to get rid of those shortly.